Portugues.json

```
@@ -0,0 +1,159 @@
+                "upcoming": "Se aproximando \u23f3",
+                "expired": "Expirado \u274c",
+                "claimed": "Coletado \u2714",
+                "ready_to_claim": "Pronto para coleta \u23f3"
+            },
+            "starts": "Comeca em: {time}",
+            "ends": "Termina em: {time}",
+            "allowed_channels": "Canais Autorizados:",
+            "all_channels": "Todos",
+            "and_more": "e {amount} mais...",
+            "percent_progress": "{percent} de {minutes} minutos",
+            "minutes_progress": "{minutes} minutos"
+        },
+        "settings": {
+            "general": {
+                "name": "Geral",
+                "autostart": "Auto-Inicio: ",
+                "tray": "Auto-iniciar na bandeija de tarefas: ",
+                "priority_only": "Apenas com Prioridade: ",
+                "proxy": "Proxy (Nessesita Reinicio do App):"
+            },
+            "game_name": "Nome do jogo",
+            "priority": "Prioridade",
+            "exclude": "Excluir",
+            "reload": "Recarregar",
+            "reload_text": "Maioria das mudancas requer o reinicio para que as mudancas sejam aplicadas: "
+        },
+        "help": {
+            "links": {
+                "name": "Links Uteis",
+                "inventory": "Ver inventario da Twitch",
+                "campaigns": "Veja todas campanhas e gerencie conecoes da sua conta"
+            },
+            "how_it_works": "Como funciona",
+            "how_it_works_text": "Cada apriximadamente 60 segundos, a aplicacao mandaum \"minute watched\" ao canal com o evento quie esta sendo assistido - isso e o bastante para o sistema registrar os drops. Note que esse modo circula o modo de precissar carregar a stream e som. para deixar o status: (ONLINE or OFFLINE) dos canais atualizados, existe uma conecao estilo websocket estabelecida que coordena quando a stream esta no ar ou fora do ar,ou atualiza o numero de visualizadores ao vivo.",
+            "getting_started": "Iniciando",
+            "getting_started_text": "1. Insira as informacoes da Twitch.\n2. Certifiquise que todas campanhas estao conectadas com as contas dos respectivos aplicativos.\n3. se quer minerar tudo clique em, descelecionar \"Priority only\" e clique em \"Reload\".\n4. Se voce quer minerar certos jogos antes, use a lista \"Priority\" para configurar a lista de jogos que voce quer minerar. A lista segue em ordem de primeri ao ultimo, sendo assim, a mineracao continuara para os jogos seguentes depois que o de cima esteja completo.\n5. Apenas deixe a opcao \"Priority only\" ativa, para evitar que minere jogos que nao sejam da sua vontade. Ou nao - voce quem manda.\n6 .Use a funcao  \"Exclude\" para excluir jogos que NUNCA sejam minerados.\n7. qualquer mudanca das listas peuqena ou grande, ou mudar a lista de \"Priority only\", requer que voce pressione \"Reload\" para que as mudancas tenham efeito."
+        }
+    }
+}
```

Missing new line at the end of the file.

```suggestion
}

```